vocabularies/Org.OData.Core.V1.xml

```diff
@@ -510,9 +510,13 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
       </Term>
       <ComplexType Name="OptionalParameterType">
         <Property Name="DefaultValue" Type="Edm.String" Nullable="true">
-          <Annotation Term="Core.Description" String="Default value for an optional parameter of primitive or enumeration type, using the same rules as the `cast` function in URLs." />
+          <Annotation Term="Core.Description" String="Default value for an optional parameter of primitive or enumeration type, using the same rules as the `cast` function in URLs" />

```

Comment:
```suggestion
          <Annotation Term="Core.Description" String="Default value for an optional parameter of primitive type, using the same rules as the `cast` function in URLs" />
```
[OData Protocol, chapter 3 Data Model](https://docs.oasis-open.org/odata/odata/v4.02/csd01/part1-protocol/odata-v4.02-csd01-part1-protocol.html#DataModel) states that
> Enumeration types are named primitive types whose values are named constants with underlying integer values.